project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementValidationService.java

```diff
@@ -102,7 +103,8 @@ public ValidationResult validateNGS(MeasurementUpdateInformationNGS update, Proj
   }
 
   @PreAuthorize("hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'WRITE')")
-  public ValidationResult validatePxp(MeasurementRegistrationInformationPxP registration, ProjectId projectId) {
```

Comment:
Please format only files you touched. It is very cumbersome to review changes due to formatting in files that were not changed.